projects/ffmpeg.org/package.yml

```diff
@@ -20,6 +20,8 @@ provides:
 dependencies:
   lame.sourceforge.io: '>=3.98.3'
   libsdl.org: ^2
+  freetype.org: "*"
+  harfbuzz.org: "*"
```

Comment:
due to the way linking works, library dependencies should be specified as `^{{major}}` if there's no tighter requirement.